plugins/services/src/js/structs/__tests__/Framework-test.js

```diff
@@ -85,6 +85,33 @@ describe("Framework", function() {
 
       expect(service.getFrameworkName()).toEqual(undefined);
     });
+
+    it("returns correct full name for service comparison with slash", function() {
```

Comment:
You are correct, it slipped through. I actually removed those test but included some tests for the getTasks and getCompletedTasks